runtime/oti/j9nonbuilder.h

```diff
@@ -5022,6 +5022,9 @@ typedef struct J9InternalVMFunctions {
 	void (*releaseVThreadInspector)(struct J9VMThread *currentThread, jobject thread);
 #endif /* JAVA_SPEC_VERSION >= 19 */
 	UDATA (*checkArgsConsumed)(struct J9JavaVM * vm, struct J9PortLibrary* portLibrary, struct J9VMInitArgs* j9vm_args);
+#if defined(J9VM_ZOS_3164_INTEROPERABILITY) && (JAVA_SPEC_VERSION >= 17)
+	I_32 (*invoke31BitJNI_OnXLoad)(struct J9JavaVM *javaVM, void *handle, jboolean isOnLoad, void *reserved);
```

Comment:
Suggest the first parameter should be named `vm` for consistency, here and and in related places.

Reply:
I've updated the first parameter name to `vm`.